src/Google.Api.Gax/ApiCallExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Grpc.Core;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Google.Api.Gax
+{
+    static class ApiCallExtensions
```

Comment:
Nit - presumably you want this to be internal (given that it's not public). Let's make that explicit.